### PR TITLE
[1LP][RFR] expelling remaining quadicons from tests and classes

### PR DESCRIPTION
**Pull request by izapolsk**
{{pytest: cfme/tests/cloud/test_stack.py cfme/tests/control/test_bugs.py cfme/tests/infrastructure/test_advanced_search_vms.py cfme/tests/storage/test_object_store.py cfme/tests/cloud/test_delete_cloud_object.py cfme/tests/cloud_infra_common/test_tag_objects.py cfme/tests/configure/test_access_control.py cfme/tests/configure/test_default_views_cloud.py cfme/tests/configure/test_default_views_infra.py cfme/tests/infrastructure/test_host.py cfme/tests/openstack/cloud/test_instances.py cfme/tests/openstack/infrastructure/test_host_power_control.py cfme/tests/openstack/infrastructure/test_hosts.py}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit ef699625bbb3f39384d02f55509078aa7b9850ca:

`cfme/tests/configure/test_default_views_infra.py`:
- :bangbang: Line 66:14: **E225** *missing whitespace around operator*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f8e15d960fb2cee3d48a647c293037287534ba4f:

`cfme/tests/configure/test_default_views_infra.py`:
- :bangbang: Line 66:14: **E225** *missing whitespace around operator*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2dc262db3ac5719656a0551cdd234d2d253ea9b9:

`cfme/tests/configure/test_default_views_infra.py`:
- :bangbang: Line 66:14: **E225** *missing whitespace around operator*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 376436a9fd051a1a5f6d05d386b01022fa6b776a:

`cfme/tests/configure/test_default_views_infra.py`:
- :bangbang: Line 66:14: **E225** *missing whitespace around operator*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a62d09643b9edac93b8956c033a101e34acb281b:

`cfme/tests/openstack/cloud/test_instances.py`:
- :heavy_exclamation_mark: Line 178:1: **W293** *blank line contains whitespace*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/configure/test_access_control.py [hunk 1]
```diff
@@ -518,13 +518,14 @@ def _test_vm_provision():
...
 
 
-def _test_vm_power_on():
-    """Ensures power button is shown for a VM"""
-    logger.info("Checking for power button")
-    vm_name = vms.Vm.get_first_vm_title()
-    logger.debug("VM " + vm_name + " selected")
-    if not vms.is_pwr_option_visible(vm_name, option=vms.Vm.POWER_ON):
-        raise OptionNotAvailable("Power button does not exist")
+# this fixture is used in disabled tests. it should be updated along with tests
```
Comment (mshriver) [inline cfme/tests/configure/test_access_control.py]: :+1:

**Comment by mshriver** [comment]
Test results are mixed, but the overall pass rate isn't far off from the complete tests. I don't want to muddy this laser focused PR with random test tweaks/fixes.

**Pull request merged**